src/controller/tests/data_model/TestCommands.cpp

```diff
@@ -116,27 +115,32 @@ void DispatchSingleClusterCommand(const ConcreteCommandPath & aCommandPath, chip
         }
         else if (responseDirective == kSendMultipleSuccessStatusCodes)
         {
+            apCommandObj->AddStatus(aCommandPath, Protocols::InteractionModel::Status::Success,
+                                    "No error but testing status success case");
+
             // TODO: Right now all but the first AddStatus call fail, so this
             // test is not really testing what it should.
-            for (size_t i = 0; i < 4; ++i)
+            for (size_t i = 0; i < 3; ++i)
```

Comment:
Why the bounds change?

Reply:
Tried to make the code equivalent. Realistically it does not matter ...